docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
             - ./otter:/app/otter
             - ./scripts:/app/scripts
             - ./twisted:/app/twisted
+            - ./schema:/app/schema
```

Comment:
Yeah. That is true. However, my (intial?) goal for docker was to primarily set up dev environment. Doing deploys via docker are not gonna happen anytime soon.